utils/send_email.py

```diff
@@ -78,6 +95,12 @@ def get_user_emails_by_tax_payer_id(tax_payer_id):
     return list(emails)
 
 
+def get_buyer_emails_by_tax_payer_id(tax_payer_id):
```

Comment:
nit: add some typing to this sign 
`def get_buyer_emails_by_tax_payer_id(tax_payer_id: int) -> List[str]:`